entries/m/microworkers.com.json

```
@@ -0,0 +1,14 @@
+{
+  "Microworkers": {
+    "domain": "microworkers.com",
+    "url": "https://www.microworkers.com",
+    "tfa": [
+      "totp",
+      "email"
+    ],
+    "documentation": "https://www.microworkers.com/blog/whats-new-in-mw-implementation-of-two-factor-authentication-2fa/",
+    "categories": [
+      "task"
```

```suggestion
      "other"
```

The `task` category is for task management and to-do services like Airtable and Todoist. I don't think any of our existing categories fit Microworkers well.

---

I was indeed struggling with the categorization of microworkers.com, but thanks for correcting me, I will open a issue later if I am unsure!

---

@Yang-Nankai I think you forgot to commit this change.

---

@Carlgo11 Sorry! I'm not familiar with GitHub yet! Thanks for your reminder, I should have revised and committed it now!